packages/cms/config.js

```diff
@@ -1,14 +1,16 @@
 // Return draft content from Dato app is in preview mode
 // Default to production endpoint if undefined to avoid unexpectedly exposing draft content
-let url = process.env.HASHI_DATO_ENVIRONMENT
-  ? `https://graphql.datocms.com/environments/${process.env.HASHI_DATO_ENVIRONMENT}`
-  : 'https://graphql.datocms.com'
-
-if (process.env.HASHI_ENV === 'preview') url += '/preview'
-
+// https://www.datocms.com/docs/content-delivery-api/api-endpoints
+const url = 'https://graphql.datocms.com'
 const token = process.env.HASHI_DATO_TOKEN || '2f7896a6b4f1948af64900319aed60'
+const isPreview = process.env.HASHI_ENV === 'preview'
+const datoEnvironment = process.env.HASHI_DATO_ENVIRONMENT
 
 module.exports = {
   url,
-  headers: { Authorization: token },
+  headers: {
+    Authorization: token,
+    ...(isPreview && { 'X-Include-Drafts': 'true' }),
+    ...(datoEnvironment && { 'X-Environment:': datoEnvironment }),
```

Comment:
I think this will fix your error!

```suggestion
    ...(isPreview && { 'X-Include-Drafts': 'true' }),
    ...(datoEnvironment && { 'X-Environment': datoEnvironment }),
```

Reply:
hah, busted. Thank you :)